### ISSUE TITLE
A fatal error has been detected by the Java Runtime Environment

### ISSUE THREAD

**Issue by infinite-dev22** [oracle/graal#6954]
## Description:
Am building a JavaFX application using MaterialFX library. When I click on a ComboBox and open it, select a value then open it again, the error occurs then. JavaFX stacktrace says ```The window has already been closed``` then next is ```A fatal error has been detected by the Java Runtime Environment:``` message from GraalVM.

## GraalVM and My environment:
 - GraalVM version 22.3.0 (19.0.1+10) (build 19.0.1+10-jvmci-22.3-b08)
 - JDK major version: 17
 - OS: Ubuntu 23.04 (Linux Kernel 6.4.0-060400-generic)
 - Architecture: AMD64

## More details
```
    Exception in thread "JavaFX Application Thread" java.lang.IllegalStateException: The window has already been closed
	at javafx.graphics@20.0.1/com.sun.glass.ui.Window.checkNotClosed(Window.java:293)
	at javafx.graphics@20.0.1/com.sun.glass.ui.Window.getNativeWindow(Window.java:318)
	at javafx.graphics@20.0.1/com.sun.glass.ui.gtk.GtkWindow.getNativeWindow(GtkWindow.java:181)
	at javafx.graphics@20.0.1/com.sun.prism.PresentableState.update(PresentableState.java:368)
	at javafx.graphics@20.0.1/com.sun.javafx.tk.quantum.SceneState.update(SceneState.java:104)
	at javafx.graphics@20.0.1/com.sun.javafx.tk.quantum.GlassScene.updateSceneState(GlassScene.java:285)
	at javafx.graphics@20.0.1/com.sun.javafx.tk.quantum.GlassScene.releaseSynchronization(GlassScene.java:142)
	at javafx.graphics@20.0.1/javafx.scene.Scene$ScenePulseListener.pulse(Scene.java:2631)
	at javafx.graphics@20.0.1/com.sun.javafx.tk.Toolkit.lambda$runPulse$2(Toolkit.java:398)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics@20.0.1/com.sun.javafx.tk.Toolkit.runPulse(Toolkit.java:397)
	at javafx.graphics@20.0.1/com.sun.javafx.tk.Toolkit.firePulse(Toolkit.java:427)
	at javafx.graphics@20.0.1/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:592)
	at javafx.graphics@20.0.1/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:572)
	at javafx.graphics@20.0.1/com.sun.javafx.tk.quantum.QuantumToolkit.pulseFromQueue(QuantumToolkit.java:565)
	at javafx.graphics@20.0.1/com.sun.javafx.tk.quantum.QuantumToolkit.lambda$runToolkit$11(QuantumToolkit.java:352)
	at javafx.graphics@20.0.1/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at javafx.graphics@20.0.1/com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at javafx.graphics@20.0.1/com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:316)
	at java.base/java.lang.Thread.run(Thread.java:1589)
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f5ce294574a, pid=33760, tid=33787
#
# JRE version: OpenJDK Runtime Environment GraalVM CE 22.3.0 (19.0.1+10) (build 19.0.1+10-jvmci-22.3-b08)
# Java VM: OpenJDK 64-Bit Server VM GraalVM CE 22.3.0 (19.0.1+10-jvmci-22.3-b08, mixed mode, tiered, jvmci, jvmci compiler, compressed oops, compressed class ptrs, g1 gc, linux-amd64)
# Problematic frame:
# V  [libjvm.so+0xd4574a]  JVM_handle_linux_signal+0x13a
#
# Core dump will be written. Default location: Core dumps may be processed with "/usr/share/apport/apport -p%p -s%s -c%c -d%d -P%P -u%u -g%g -- %E" (or dumping to /home/infinite/IdeaProjects/spoty/core.33760)
#
# An error report file with more information is saved as:
# /home/infinite/IdeaProjects/spoty/hs_err_pid33760.log
#
# The crash happened outside the Java Virtual Machine in native code.
```
[hs_err_pid33760.log](https://github.com/oracle/graal/files/11987375/hs_err_pid33760.log)

**Comment by oubidar-Abderrahim** [comment]
Hi thank you for reporting this issue, 

> GraalVM version 22.3.0 (19.0.1+10) (build 19.0.1+10-jvmci-22.3-b08)
JDK major version: 17

First, please always use the latest version. for GraalVM 22 that's 22.3.2. second, build 19.0.1 means this is JDK 19, not JDK 17, please make sure to specify the correct JDK used.

Please test using 22.3.2 for JDK17 and let us know your findings. Thank you.

**Comment by infinite-dev22** [comment]
Fixed, Tested using GraalVM Oracle 17.0.7-graal(Installed using SDKMan) and no exception thrown. Thanks for the help.

![Screenshot from 2023-07-15 05-24-23](https://github.com/oracle/graal/assets/77667007/1fcc84bc-f353-424d-916d-c5d3e5b8afe7)

**Comment by barbarosalp** [comment]
I have the same issue with the latest version of Graal VM from Oracle
